index.html

```diff
@@ -1259,6 +1267,54 @@ <h1 class="text-white">FOSSASIA is the annual OpenTechSummit featuring prominent
         <script src="js/tweetsfetcher.js"></script>
         <script src="js/placeholders.min.js"></script>
         <script src="js/scripts.js"></script>
+        <script>
+        $.ajax({
+	        type : 'GET',
+	        dataType : 'json',
+	        url: 'schedule/speakers.json',
```

Comment:
Please fetch the speaker list from https://raw.githubusercontent.com/fossasia/open-event-scraper/master/out/speakers.json